clients/cobol-lsp-vscode-extension/src/test/suite/lsp.spec.xmlgenerate.test.ts

```diff
@@ -0,0 +1,364 @@
+/*
+ * Copyright (c) 2025 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+
+import * as assert from "assert";
+import * as helper from "./testHelper";
+import { pos } from "./testHelper";
+import * as vscode from "vscode";
+import * as path from "path";
+
+suite("TF48417: XML GENERATE", function () {
+  suiteSetup(async function () {
+    this.timeout(0);
+    this.slow(2000);
+    await helper.updateConfig("basic.json");
+    await helper.activate();
+  });
+
+  this.afterEach(async () => await helper.closeAllEditors()).timeout(
+    helper.TEST_TIMEOUT,
+  );
+
+  this.afterAll(async () => await helper.closeAllEditors()).timeout(
+    helper.TEST_TIMEOUT,
+  );
+
+  test("TC369351: Identifier1 must be alphanumeric", async () => {
+    const extSrcPath = path.join("TEST2.CBL");
+    const diagPromise = helper.waitForDiagnosticsChange(extSrcPath);
+    await helper.showDocument(extSrcPath);
+    const editor = helper.getEditor("TEST2.CBL");
+    let diagnostics = await diagPromise;
+    await helper.insertString(
+      editor,
+      pos(19, 0),
+      "       01 XML-DOC123 PIC X(5000) USAGE NATIONAL.\n",
+    );
+    await helper.deleteLine(editor, 23);
```

Comment:
The statement is not related, however it should be be deleted because of TEST2.cbl content. which has been selected that It is the closest one to the template. Consider creating a template to remove these operations.